xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/VFS.h

```diff
@@ -135,9 +138,9 @@ extern "C"
 
   typedef struct AddonInstance_VFSEntry /* internal */
   {
-    AddonProps_VFSEntry props;
-    AddonToKodiFuncTable_VFSEntry toKodi;
-    KodiToAddonFuncTable_VFSEntry toAddon;
+    AddonProps_VFSEntry* props;
+    AddonToKodiFuncTable_VFSEntry* toKodi;
+    KodiToAddonFuncTable_VFSEntry* toAddon;
```

Comment:
what about std::unique ptrs?

Reply:
We can not use a C++ `std::unique` in a "C" ABI, thats one of the main reason of this request.

Reply:
oh yea, didn't look at the location of this file.

Reply:
NP, thanks a lot for review!